winterfell/webapp/src/js/claimSelectFormsCtrl.js

```diff
@@ -1,13 +1,65 @@
 var app = angular.module('vetafiApp');
-app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', 'formTemplateService', '$stateParams',
-  function($scope, claimService, formTemplateService, $stateParams) {
+app.controller('claimSelectFormsCtrl', ['$scope', '$state', '$stateParams', 'claimService', 'formTemplateService',
+  function($scope, $state, $stateParams, claimService, formTemplateService) {
     $scope.claimId = $stateParams.claimId;
-    $scope.forms = formTemplateService;
+
+    $scope.allForms = formTemplateService; // returns an object {formId -> {...}}
+    var userForms = [ // TODO: state of all forms edited by user
```

Comment:
That is correct, the `formTemplateService` is supposed to be immutable, given a form id, it gives you the template which allows you to construct the list of questions and present them to the user (the name `formTemplateService` might be bad, but its just what I randomly thought to call it).

What you want is the state of current forms. I feel like that is something that should be stored in the database and retrieved for a given claim id/user id combo.